src/Data/OpenApi.hs

```diff
@@ -412,7 +412,6 @@ import Data.OpenApi.Internal
 --     ],
 --     "type": "object"
 -- }
-
```

Comment:
this empty line is crucial, it separates a new __section__ "manipulation" from the previous one. please bring it back :)